modules/ROOT/pages/proxy-domain-support.adoc

```diff
@@ -1,10 +1,10 @@
 = Domain Support for Mule 4 API Proxies
 
-Mule runtime engine enables you to share common resources among other Mule applications by defining these resources within a Mule domain. Other Mule applications can then inherit this domain.
+Mule runtime engine (Mule) enables you to share common resources among other Mule applications by defining these resources within a Mule domain. Other Mule applications can then inherit this domain.
 
 This way, your Mule applications that are configured for it, can access the xref:4.1@mule-runtime::shared-resources.adoc[resources]declared in your domain. +
 
-Because every auto-generated proxy is a preconfigured Mule application, you can also define multiple API proxies under the same domain. You install a domain when you auto-deploy multiple proxies that use the same host and port but have a different path. +
+Because every auto-generated proxy is a preconfigured Mule application, you can define multiple API proxies under the same domain. You create a domain when you auto-deploy multiple proxies that use the same host and port but have a different path. +
```

Comment:
I don't understand the 2nd sentence very clearly. Is it saying that the only way to create a domain is to auto-deploy multiple proxies that use the same host & port but have a different path (do we mean different endpoints? Base URLs?

Reply:
path is a field in the UI and it is part of creating a proxy domain. Thanks!

Reply:
Done! Thanks!